src/Tizen.NUI/src/internal/Interop/Interop.QuickPanelClient.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.NUI
+{
+    internal static partial class Interop
+    {
+        internal static partial class QuickPanelClient
+        {
+            const string lib = "libtzsh_quickpanel.so.0";
+
+            [global::System.Runtime.InteropServices.DllImport(lib, EntryPoint = "tzsh_quickpanel_create_with_type")]
+            internal static extern IntPtr tzshQuickPanelCreateWithType(IntPtr tzsh, IntPtr win, int type);
+
+            [global::System.Runtime.InteropServices.DllImport(lib, EntryPoint = "tzsh_quickpanel_destroy")]
+            internal static extern int tzshQuickPanelDestroy(IntPtr qpClient);
```

Comment:
When the method returns the errorcode, I recommend use ErrorCode enum type instead of int as return type.
```suggestion
            internal static extern ErrorCode tzshQuickPanelDestroy(IntPtr qpClient);
```
C# P/Invoke will convert the return value to ErrorCode from int automatically.
If you use enum type directly, you can remove many castings in your code.

see ref: https://github.com/Samsung/TizenFX/blob/abcae5c9b69e60daccf6e5096ed1be1623922b12/src/Tizen.Applications.WidgetApplication/Interop/Interop.Widget.cs#L88

Reply:
ErrorCode is defined in Interop.TizenShell.cs
So in Interop.QuickPanelService.cs and Interop.QuickPanelClient.cs change code like this.

[global::System.Runtime.InteropServices.DllImport(lib, EntryPoint = "tzsh_quickpanel_service_destroy")]
            internal static extern Interop.TizenShell.ErrorCode Destroy(IntPtr service);

So is it okay using interop codes in other interop.cs ?